base/bucket_gocb.go

```diff
@@ -250,7 +251,7 @@ func (bucket *CouchbaseBucketGoCB) retrievePurgeInterval(uri string) (int, error
 
 	// Server purge interval is a float value, in days.  Round up to hours
 	purgeIntervalHours := int(purgeResponse.PurgeInterval*24 + 0.5)
-	return purgeIntervalHours, nil
+	return time.Duration(purgeIntervalHours) * time.Hour, nil
```

Comment:
If we're still rounding this up to hours, what's the advantage in switching the stored value from an int value to a duration?